### PR TITLE
Configure mysql docker container

### PR DESCRIPTION
**Pull request by mwarkentin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,47 @@
-# mysql
-Convox-maintained MySQL Docker image
+# convox/postgres
```
Comment (ddollar) [inline README.md:1]: The README still says postgres in a few places

**Comment by ddollar** [comment]
Thanks :)

**Comment by mwarkentin** [comment]
No problem! Are you able to publish this image as is? Maybe as an RC or beta?

**Comment by ddollar** [comment]
https://hub.docker.com/r/convox/mysql/

**Pull request merged**